fastify.js

```diff
@@ -59,6 +59,28 @@ const onBadUrlContext = {
   onError: []
 }
 
+const caseInsensitiveParse = (string) => {
+  const original = querystring.parse(string)
+  const parsed = Object.keys(original).reduce((acc, key) => {
+    const lowerKey = key.toLowerCase()
+    const existingItem = acc[lowerKey]
+    const originalValue = original[key]
+
+    // If a variation of the key exists, merge both
+    if (existingItem) {
+      // Flat array
+      acc[lowerKey] = [existingItem, originalValue].reduce((acc, val) => acc.concat(val), [])
```

Comment:
This seems surprising and should be noted in the documentation. How is this different, if it is, from the way core `querystring.parse` handles same keys?

Reply:
Yes, it's working the same as the default core: https://www.npmjs.com/package/query-string
![image](https://user-images.githubusercontent.com/7080621/98381740-4bec1880-2018-11eb-9b84-3de7a4305e22.png)